src/command/root-command/command-log.ts

```diff
@@ -127,4 +129,15 @@ export class CommandLog {
 
     return result.value
   }
+
+  public printError(error: ErrorWithStatus, notFoundMessage: string): void {
```

Comment:
I suggest that you use `printBeeError` instead to clearly distinguish from other error messages.

Reply:
Also I would put notFoundMessage to an optional `options` object, and if `notFoundError` happens, use that one or the error message from the response.